src/Castle.Core/DynamicProxy/IInterceptorBase.cs

```diff
@@ -0,0 +1,56 @@
+﻿
+namespace Castle.DynamicProxy
+{
+	using System;
+
+	public class ExceptionContext
+	{
+
+		public Exception Ex { get; set; }
+		public IInvocation Invocation { get; set; }
+	}
+
+	public abstract class InterceptorBase : IInterceptor
```

Comment:
I find your interceptor to actually have a fairly specific logic (which unfortunately isn't documented at all e.g. in XML doc comments), and for that this name seems to generic. And there's my main issue with this PR: The interceptor seems at the same time too specific for a low-level infrastructure library like DynamicProxy, and short enough to be easily written in your own application code.